xrpl4j-binary-codec/src/main/java/org/xrpl/xrpl4j/codec/binary/types/AmountType.java

```diff
@@ -27,8 +27,8 @@
   public static final int NATIVE_AMOUNT_BYTE_LENGTH = 8;
   public static final int CURRENCY_AMOUNT_BYTE_LENGTH = 48;
   private static final int MAX_IOU_PRECISION = 16;
-  private static final int MIN_IOU_EXPONENT = -96;
-  private static final int MAX_IOU_EXPONENT = 80;
+  private static final int MIN_IOU_EXPONENT = -81;
```

Comment:
Maybe add a Javadoc note here with a link to xrpl.org explaining why these don't match -96 and 80?